src/components/DopplerPlus/AgenciesForm.js

```diff
@@ -0,0 +1,272 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+import { InjectAppServices } from '../../services/pure-di';
+import { Field } from 'formik';
+import {
+  EmailFieldItem,
+  FieldGroup,
+  FieldItem,
+  FormWithCaptcha,
+  InputFieldItem,
+  PhoneFieldItem,
+  SubmitButton,
+} from '../form-helpers/form-helpers';
+import { useIntl } from 'react-intl';
+import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
+
+const AgenciesForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  // TODO: apply content
+  // TODO: add check to accept privacy policies
+
+  const volumeOptions = [
+    {
+      id: 'lessThan500k',
```

Comment:
Why not add all this options inside resource file? Check that `value` and `description` are the same. 🤔

Reply:
When you say resource file you mean utils.js? 

The value not necessarily has to be equal to the description. The description will be a translation.